docs/en/administrator-guide/config/be_config.md

```diff
@@ -30,51 +30,70 @@ under the License.
 
 This document mainly introduces the relevant configuration items of BE.
 
+The BE configuration file `be.conf` is usually stored in the `conf/` directory of the BE deployment path. In version 0.14, another configuration file `be_custom.conf` will be introduced. The configuration file is used to record the configuration items that are dynamically configured and persisted by the user during operation.
```

Comment:
... is used for recording ... ?

Reply:
This is translated by Google Translator... So, I think it is correct ^^